packages/celotool/ci_test_sync_with_network.sh

```diff
@@ -56,9 +56,10 @@ geth_tests/network_sync_test.sh ${NETWORK_NAME} full && echo
 geth_tests/network_sync_test.sh ${NETWORK_NAME} light && echo
 test_ultralight_sync ${NETWORK_NAME} && echo
 
-export NETWORK_NAME="alfajoresstaging"
-geth_tests/network_sync_test.sh ${NETWORK_NAME} full && echo
+#TODO(Kobi): disabled until alfajoresstaging is upgraded with BLS
```

Comment:
Please create an issue and add an issue number here as well to re-enable these tests.

Reply:
Should I add the issue number in the code?